app.rb

```diff
@@ -3,13 +3,15 @@
 require_relative './book'
 require_relative './rental'
 
+require 'json'
+
 class App
   attr_accessor :persons, :books, :rentals
 
   def initialize()
-    @persons = []
-    @books = []
-    @rentals = []
+    @persons = [JSON.parse(File.read('persons.json'))]
+    @books = [JSON.parse(File.read('books.json'))]
+    @rentals = [JSON.parse(File.read('rentals.json'))]
   end
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/203591362-8af98e59-d725-409d-8c5e-4093297d288d.png)

- [ ] When we start the application without having data json files then your application shows this error shown above :arrow_up: so your application should see that if book.json file or rental.json or person.json file doesn't exist then your application should create these files before initializing the application :blush: